assets/src/scripts/outputs-viewer/components/FileList/FileList.jsx

```diff
@@ -29,7 +30,7 @@ function FileList() {
     // If there are horizontal scrollbars, minus 17px for the scrollbar
     const fileListHeight =
       window.innerHeight -
-      listEl.current?.getBoundingClientRect().top -
+      (listEl.current?.getBoundingClientRect().top || 0) -
       30 -
       (hasScrollbarX ? 17 : 0);
```

Comment:
Might be worth pulling some of these calculations out into separate values to help with readability?  The comment is great but I find myself jumping back and forth between the magic numbers and the explanation!

Reply:
Good point, added some more in.